### DISCUSSION TITLE
should `InlineAlert` have a `Header` or a `Heading` (or both)?

### DISCUSSION THREAD

**Discussion by stefanprobst** [adobe/react-spectrum#4823] [Q&A] [answered]
it seems the spectrum docs say there should be "an extra-extra-small heading component" (see https://spectrum.adobe.com/page/in-line-alert/#Title-area), but the current react-spectrum implementation wants a `Header`, not a `Heading`?

**Comment by reidbarber** [comment]
I think this is a good point. Heading is for text, whereas Header is more of a container. ~I'll convert this into an issue.~ Thanks for catching!

**Update**: On the other hand, InlineAlert has Content which is a container, so maybe it makes more since to pair that with Header? I'll bring this up with the team.

**Reply by reidbarber** [reply, marked as answer]
Looks like this is the correct change. I've made a PR here: https://github.com/adobe/react-spectrum/pull/4834
